apps/adapt-essentials-asset-fields/src/locations/ConfigScreen.tsx

```diff
@@ -2,17 +2,34 @@ import { useCallback, useEffect, useState } from 'react';
 import { ConfigAppSDK } from '@contentful/app-sdk';
 import { useSDK } from '@contentful/react-apps-toolkit';
 import MarkdownRender from '../components/MarkdownRender';
-import { Box } from '@contentful/f36-components';
+import { Box, Text } from '@contentful/f36-components';
 import { Workbench } from '@contentful/f36-workbench';
+import { css } from 'emotion';
 
 import originalReadme from '../../README.md?raw';
 
 export interface AppInstallationParameters {}
 
+const InstallationInstructions = () => (
+  <Box>
+    <h2
+      className={css({
+        margin: '1em 0',
+      })}>
+      Installation
+    </h2>
+    <Text>
+      To <b>install</b> the App click install button on the right bellow the settings menu.
```

Comment:
Typo: change `bellow` to `below`

Reply:
🙈 Thanks!
Fix https://github.com/contentful/marketplace-partner-apps/pull/998/commits/549c506544e51980b23ccd731fb5ee415f3893df